index.html

```diff
@@ -1932,6 +1927,43 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code>splashscreens</code> member
+        </h3>
+        <p>
+          A <dfn>splash screen</dfn> is a graphical representation (usually an
+          image on a background color) that is presented as the end user before
+          the user agent is able render content. A splash screen is intended to
+          convey to the end-user that the web application is being prepared for
+          them in the background.
+        </p>
+        <p>
+          When showing a <a>splash screen</a>, a user agent MUST announce to
+          assistive technologies that the web application is being loaded in
+          the background.
```

Comment:
"a user agent MUST announce to assistive technologies" is unclear, suggest something in form of "users agents MUST expose the loading state of the application via the platform <a>Accessibility API</a>(s)"